charts/pega/values-minimal.yaml

```diff
@@ -0,0 +1,89 @@
+---
+global:
```

Comment:
Why are only some of the parameters under global? Is that correct? `docker` is in both.